napari/layers/shapes/_shape_list.py

```diff
@@ -82,7 +84,9 @@ class ShapeList:
         be rendered.
     """
 
-    def __init__(self, data=(), ndisplay=2) -> None:
+    def __init__(
+        self, data: typing.Iterable[Shape] = (), ndisplay: int = 2
```

Comment:
For my continuing education, why not import Iterable with the other typing imports?

Reply:
I tried this, but there's already a `from abc import Iterable` in the import block, so to avoid conficts I had to use `typing.Iterable`. Maybe it would be better the other way round though (use `abc.Iterable` instead)